sample/notebook_with_metadata.jl

```diff
@@ -0,0 +1,30 @@
+### A Pluto.jl notebook ###
+# v0.16.1
+
+using Markdown
+using InteractiveUtils
+
+# This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
+macro bind(def, element)
+    quote
+        local el = $(esc(element))
+        global $(esc(def)) = Core.applicable(Base.get, el) ? Base.get(el) : missing
+        el
+    end
+end
+
+# ╔═╡ 7d550d46-7dc4-11eb-3df0-db7ad0594043
+# ╠═╡ disabled = "directly"
+using PlutoUI
+
+# ╔═╡ 8399ed34-7dc4-11eb-16ec-e572834e149d
+# ╠═╡ disabled = "indirectly"
```

Comment:
Probably there is a good reason for this, but can't the indirectly disabled be derived from the topology? Or is the topology disabled when a cell is disabled?

Reply:
The indirectly status is derived from topology when the notebook is executed in Pluto. But the notebook.jl file should also run as a stand-alone Julia script. For this purpose, cells dependent on deactivated cells must be commented out, too.

The sample notebook, however, needs to be updated because it does not have the disabled cells commented out yet.

Reply:
> For this purpose, cells dependent on deactivated cells must be commented out, too.

Nice!

Reply:
Sample notebook is now updated.

The slider is now deactivated, not the `using PlutoUI` cell, because Pluto cannot figure out cells dependent on using calls (therefore using calls are always executed first).

Reply:
> because Pluto cannot figure out cells dependent on using calls

That actually got fixed by Paul! https://github.com/fonsp/Pluto.jl/pull/1541 But it's a new type of reference (see the PR), which does not carry the "disabling" satus.